challenges/objects-arrays.js

```diff
@@ -59,11 +94,24 @@ Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+
+graduates.map((grads, i) => {
+  contactInfo[i] = `${grads.first_name} ${grads.email}`
+})
+
+
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+
+let a = graduates.filter(target => target.university.includes('Uni'))
+
+a.map((o,i) => {
+  uni[i] = o.university
```

Comment:
You're doing a great job reading the MDN docs and seeing that you can use a second parameter in the cb to reference the index.  However, what is the implication in this example you game me?  Let's say you have a `graduates` array with 10 elements.  Only the last element contains `Uni`.  What would your `uni` array look like?

Reply:
This sounded like a trick but originally I thought it would just put the last element into the array being that it would be the only one with `uni` inside of the element. I changed the names of the universities in the original array to make sure and it only logged the last one like I had thought?